src/classes/UTIL_Describe.cls

```diff
@@ -496,6 +498,31 @@ public class UTIL_Describe {
 
         return picklistValues;
     }
+    
+    /****
+     * @description Returns a map of active, user-available fields labels and api names for a given
+     * SObjectType and field type, keyed by  label with a value of  value.
+     * @param objectName Name of the object to retrieve values for.
+     * @param fieldName Name of the field to retrieve values for.
+     * @return Map<String, String> Map of labels to values for the object and field specified.
+     *****/
+    @AuraEnabled
+    public static Map<String, String> getPhoneFields(String objectName, String type) {
```

Comment:
This is *very* close to the getFieldsOfType() method but is called "getPhoneFields" -- if this is specifically for getting the phone fields on the Contact, do we actually need to pass in the objectName and type here? (Is there any possible way we can just make use of the getFieldsOfType() method?)

Reply:
getFieldsOfType() returns a map which does not support @ AuraEnabaled, so was forced to created this new method.

Reply:
Okay - then I think we can make it more generic and remove the parameters.

Reply:
Not sure if I am following, we need the parameters in the method to process the fields based on data type.

Reply:
This method is called getPhoneFields() which is pretty explicit, so I wouldn't expect it to work for any other type other than Phone fields, and possibly not for any other object. Do we need the type parameter? Do we need the objectName parameter?

If we want to keep this generic, maybe it's called something like `getFieldsOfTypeForAuraComponents`?